packages/react-app-revamp/lib/contests/index.ts

```diff
@@ -28,9 +28,9 @@ async function getContractConfig(address: string, chainName: string, chainId: nu
   return contractConfig;
 }
 
-const fetchTokenBalances = async (contest: any, contestRewardModuleAddress: string) => {
+export const fetchTokenBalances = async (chainName: string, contestRewardModuleAddress: string) => {
   try {
-    const networkName = contest.network_name.toLowerCase() === "arbitrumone" ? "arbitrum" : contest.network_name;
+    const networkName = chainName.toLowerCase() === "arbitrumone" ? "arbitrum" : chainName;
     const alchemyRpc = Object.keys(alchemyRpcUrls).filter(url => url.toLowerCase() === networkName)[0];
     //@ts-ignore
     const alchemyAppUrl = `${alchemyRpcUrls[alchemyRpc]}/${process.env.NEXT_PUBLIC_ALCHEMY_KEY}`;
```

Comment:
also this and anywhere else this environment variable is used should be updated in the code base to use the new config and env var structure defined in #402.